core/services/fluxmonitorv2/store.go

```diff
@@ -0,0 +1,121 @@
+package fluxmonitorv2
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink/core/services/job"
+	"github.com/smartcontractkit/chainlink/core/services/pipeline"
+	corestore "github.com/smartcontractkit/chainlink/core/store"
+	"gorm.io/gorm"
+)
+
+//go:generate mockery --name Store --output ./mocks/ --case=underscore
+
+type Store interface {
```

Comment:
I think it'd be more inline with existing patterns to call this ORM and have it only contain methods relevant to FM tables i.e. `MostRecentFluxMonitorRoundID` `DeleteFluxMonitorRoundsBackThrough` `FindOrCreateFluxMonitorRoundStats` `UpdateFluxMonitorRoundStats`. The other ORMs are all mockable as well so you can just mock them directly in the tests. It does mean that the FluxMonitor struct would store KeyStoreInterface (should be pulled into its own package anyways and just called KeyStore), pipelineORM, jobORM and ORM as fields, but I actually like that as it explicitly shows the dependency.

Reply:
OK sure. Will make those changes

Reply:
Done